internal/util/projutil/interactive_promt_util_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package projutil
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+func TestUserInput(t *testing.T) {
+	type args struct {
+		msg     string
+		content []byte
+	}
+
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "test single user input",
```

Comment:
Could you please improve it by `should pass when xxx` or `should fail when yyy`?

Reply:
Modified the names. One of the test checks the user input and the other checks whether the command re-prompts when user does not provide input.